core/src/stored/sd_stats.cc

```diff
@@ -513,7 +513,7 @@ bool StatsCmd(JobControlRecord* jcr)
 
   if (job_statistics) {
     bool found;
-    JobControlRecord* jcr;
+    JobControlRecord* my_jcr;
```

Comment:
The jcr is only used for the loop below.  We should probably move the definition there as well (and maybe wrap it in a new scope).

Reply:
extracted into function